build-docker.sh

```diff
@@ -1,27 +1,36 @@
 #!/usr/bin/env bash
-set -e
+set -e -o pipefail
 
-CONTAINER_NAME=trezor-firmware-env.nix
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+CONTAINER_NAME=${CONTAINER_NAME:-trezor-firmware-env.nix}
+ALPINE_VERSION=${ALPINE_VERSION:-3.12.0}
+ALPINE_RELEASE=${ALPINE_RELEASE:-3.12}
+ALPINE_ARCH=${ALPINE_ARCH:-x86_64}
+ALPINE_CDN=${ALPINE_CDN:-http://dl-cdn.alpinelinux.org/alpine}
+CONTAINER_FS_URL=${CONTAINER_FS_URL:-"$ALPINE_CDN/v$ALPINE_RELEASE/releases/$ALPINE_ARCH/alpine-minirootfs-$ALPINE_VERSION-$ALPINE_ARCH.tar.gz"}
 
 TAG=${1:-master}
 REPOSITORY=${2:-local}
 PRODUCTION=${PRODUCTION:-1}
 MEMORY_PROTECT=${MEMORY_PROTECT:-1}
 
-if [ "$REPOSITORY" = "local" ]; then
-  REPOSITORY=file:///local/
+if [[ "$REPOSITORY" = "local" ]]; then
+  REPOSITORY="file:///local/"
 else
-  REPOSITORY=https://github.com/$REPOSITORY/trezor-firmware.git
+  REPOSITORY="https://github.com/$REPOSITORY/trezor-firmware.git"
```

Comment:
While I'm at it, why not removing this completely? REPOSITORY could be passed as a full git repo url.

Reply:
OK